patterns/2-structured/project-setup/communication-tooling.md

```diff
@@ -32,23 +30,23 @@ answer.
 ## Solution
 
 The host team needs to be clear on the benefit of providing company-public,
-archived, searchable, linkable communication channels that are free to subscribe
-to by anyone in the company.
+archived, searchable, linkable communication channels that anyone in the company can subscribe to.
 
 The goal when streamlining communication channels for InnerSource projects
 should be to align communication around topics, not around certain sets of
-people:
+people.
+
+A project should set up the following communication tooling:
+
+1. **a dedicated issue tracker** where structured communication, decision-making and progress tracking can happen transparently for all host team members but also for downstream users and contributors to follow. (For even for applications of the issue tracker see [Issue Tracker Use Cases](./issue-tracker.md)).
+2. **one or more public discussion channels** that come with less rigid a structure. Typically, this will be mailing lists, online forums or even archived chat channels. Usually it is enough to start with just one channel for the project. If traffic increases too much it is helpful to split discussions about project usage from discussions about project development.
```

Comment:
@MaineC With "archived channels" here you mean that the messages in this channel will be available for users even years later, right? I am asking because "archived channel" could also mean that the channel has been marked as "archived" and is no longer active.

We could reword this to make it easier to understand.

Reply:
I don't have a good idea of how to reword this. will leave as is for now.